anesthetic/plot.py

```diff
@@ -772,12 +779,13 @@ def kde_contour_plot_2d(ax, data_x, data_y, *args, **kwargs):
     sigmay = np.sqrt(kde.covariance[1, 1])
     p = cut_and_normalise_gaussian(tri.y, p, sigmay, ymin, ymax)
 
-    contours = iso_probability_contours_from_samples(p, weights=w)
+    levels = iso_probability_contours_from_samples(p, contours=levels,
+                                                   weights=w)
 
     if facecolor not in [None, 'None', 'none']:
         linewidths = kwargs.pop('linewidths', 0.5)
         cmap = kwargs.pop('cmap', basic_cmap(facecolor))
-        contf = ax.tricontourf(tri, p, contours, cmap=cmap, zorder=zorder,
+        contf = ax.tricontourf(tri, p, levels=levels, cmap=cmap, zorder=zorder,
                                vmin=0, vmax=p.max(), *args, **kwargs)
```

Comment:
This tiny change is in fact almost all that was needed to sort out the problem of having more than 2 contours, thus fixing #151. 

You were right that #114 did cause some of the issues, but the issue that I previously didn't know how to address was the ambiguity of `levels` as arg or as kwarg to `tricontour` (and not to be left out of `iso_probability_contours_from_samples`...).

Does this also solve #69?

Reply:
Excellent!

re #69, not quite -- we need to work out how best to reliably estimate marginal densities in the tails of the distribution (something which I'm convinced should be possible given the nature of nested sampling), but does require some more thinking.